css/vendors.css

```diff
@@ -12500,19 +12500,19 @@ fieldset[disabled] .btn-flat:hover {
 }
 
 .icon-check-gray {
-  background-image: url(/assets/images/icons/check_simple_gray.svg)
```

Comment:
💭 What's the motivation for the URL changes here? Ideally, this app never needs to know the path it lives under. Could a `<base>` tag help us out?

Reply:
I got 404s while running the Node server locally for these assets. Turns out they live in the Play repo [here](https://github.com/algorithmiaio/algorithmia/tree/develop/public/images/icons). I wanted to ensure that any assets the Node server needs in order to render its pages are contained within this repo. (Currently the CSS file is served via the CDN so requests to `/assets` are relative to the CDN, not this repo.)

I really like your point about removing the `/developers` part from this file and using the `baseurl` instead. That's much more flexible. Is it alright if we leave the paths as-is for now and address that piece of it when we wire up Webpack to this repo? It'll be much easier to inject dynamic values like `baseurl` once we have proper build tools in place.

Reply:
Gotcha. I hadn't realized we still had an asset dependency! My take: let's make a ticket to migrate those resources out of Play and into the dev-center, and tackle it directly after this ticket. Feel to mark with `Docs App` epic and move into this sprint. I'm ok with these paths being the way they are until we address that story.

Reply:
Alright, having now absorbed all of this, things make much more sense. I do think having a `<base>` tag will address most of these woes. Mind creating a ticket for the "Docs App" epic that captures doing our best to make all relative URLs location-agnostic? I agree the webpack setup will make this easier, but I'd like to explore if there's a potentially cheap way to do it with Jekyll as well, as this is a somewhat sharp edge I'd like to sand off.